src/HttpHeader.cc

```diff
@@ -400,9 +403,12 @@ HttpHeader::parse(const char *header_start, size_t hdrLen, Http::ContentLengthIn
         const char *field_start = field_ptr;
         const char *field_end;
 
+        const char *hasBareCr = nullptr;
```

Comment:
Depending on how you decide to address the other change requests, this should become at least

```suggestion
        const char *hasSuspiciousCr = nullptr;
```

or (my preference):

```suggestion
        const char *problem = nullptr;
```

Reply:
The comments in this old parser use the wording "bare CR". IIRC that is based on security issue documentation from many years ago. Since this is all going to be replaced before to long anyway I am leaving this with the existing terminology.

Reply:
> The comments in this old parser use the wording "bare CR".

What do you mean by "comments in this old parser"? I cannot find the word "bare" in the master:src/HttpHeader.cc.

While researching this issue, I discovered that "bare CR" is a technical term that essentially means CR not followed by LF. This term is used (without definition) by RFC 7230 and RFC 7231, for example, where its intended meaning is possible to discern from the context. The existing src/HttpHeader.cc code does not use this term, but we _can_ start using it. Thus, I am resolving this change request.